File: RoundTable Project Server/src/com/dommoni/roundtable/CustomFieldValueWrapperInt.java
package com.dommoni.roundtable;

public class CustomFieldValueWrapperInt extends CustomFieldValueWrapper
{
   private int value;
   
   public CustomFieldValueWrapperInt()
   {
   }
   
   public CustomFieldValueWrapperInt(int value)
   {
      this.value = value;
   }
   
   public int getValue() 
   { 
      return this.value; 
   }
   
   public void setValue(int value) 
   {
      this.value = value;
   }
}

File: RoundTable SDK for DotNET/src/CustomFieldValueWrapperInt.cs
using System;

namespace RoundTable
{
	[System.Xml.Serialization.SoapTypeAttribute("CustomFieldValueWrapperInt", "http://www.dommoni.com/ROUNDTABLE")]
	public class CustomFieldValueWrapperInt : CustomFieldValueWrapper 
	{
		/// <remarks/>
		public int value;
	}
}
